pyomo/contrib/multisolve/multisolve.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import division
+
+import logging
+import time
+import timeit
+
+_pathos_available = True
+try:
+    # Pathos is required because a dill-friendly multiprocessing alternative was necessary.
+    from pathos.pools import _ProcessPool as Pool
+except ImportError:
+    _pathos_available = False
```

Comment:
Why do you need pathos?  It looks like the only thing you are using from pathos is `pathos.pools._ProcessPool`, and that class isn't actually from pathos:
```python
>>> import pathos
>>> pathos.pools._ProcessPool
<class 'multiprocess.pool.Pool'>
```
If you dig into the pathos code, `_ProcessPool` is from multiprocessing if multiprocessing is available (since 2.6), otherwise it falls back on the older `processing` package.

Reply:
I thought that pathos was a reimplementation of the multiprocess pool using `dill` instead of `pickle`, but I could be wrong.

Reply:
Locally, at least, replacing the pathos import with a `Pool` import from `multiprocessing` causes a failure (I'm running cpython 3.6 on Ubuntu).